x-pack/plugin/otel-data/src/main/resources/component-templates/otel@mappings.yaml

```diff
@@ -0,0 +1,65 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Default mappings for all OpenTelemetry data streams
+  managed: true
+template:
+  mappings:
+    date_detection: false
+    dynamic: false
+    _source:
+      mode: synthetic
+    properties:
+      "@timestamp":
+        type: date
```

Comment:
Should we add a TODO comment to update to date_nanos when https://github.com/elastic/elasticsearch/issues/109352 is done?

Reply:
Sounds reasonable, added a comment.